lab-anthony/test/read-test.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const expect = require('chai').expect;
+const fs = require('fs');
+
+describe('Testing read module', function() {
+  describe('Testing read() file reading', function() {
+    it('Should error', function(done) {
```

Comment:
It's generally considered a no-no to test other people's modules--fs.readFile has been tested more thoroughly than you can, so there is no point in writing your own tests for it.